modules/preview-react/segmented-control/lib/SegmentedControlItem.tsx

```diff
@@ -0,0 +1,189 @@
+import * as React from 'react';
+
+import {colors, borderRadius, type, space} from '@workday/canvas-kit-react/tokens';
+import {createSubcomponent, styled, StyledType, useIsRTL} from '@workday/canvas-kit-react/common';
+import {Tooltip, TooltipProps} from '@workday/canvas-kit-react/tooltip';
+import {
+  BaseButton,
+  ButtonContainerProps,
+  ButtonColors,
+  ButtonSizes,
+  getMinWidthStyles,
+} from '@workday/canvas-kit-react/button';
+import {CanvasSystemIcon} from '@workday/design-assets-types';
+import {useSegmentedControlModel} from './hooks/useSegmentedControlModel';
+import {Text} from '@workday/canvas-kit-react/text';
+import {useSegmentedControlItem} from './hooks/useSegmentedControlItem';
+
+export interface ItemProps extends ButtonContainerProps {
+  /**
+   * Optionally pass index to the item. This should be done if `SegmentedControl.Item` components were created
+   * via a `Array::map` function. This index will ensure keyboard navigation works even if items are
```

Comment:
```suggestion
   * via a `[Array::map](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map)` function. This index will ensure keyboard navigation works even if items are
```

Reply:
Up to you on this change, but it's nice to link our to examples on mdn, thoughts?

Reply:
agree